frontend/src/components/NavigationComponents/NavigationBar.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/macro'
+import {Link} from "react-router-dom";
+
+export default function NavigationBar() {
+    return (
+        <NavigationWrapper>
+            <NavigationLink to="/recipes">REZEPT ÜBERSICHT</NavigationLink>
+            <NavigationLink to="/">CORONA LAGE</NavigationLink>
+            <NavigationLink to="/">MEIN BEREICH</NavigationLink>
+        </NavigationWrapper>
+
+    )
+}
+
+const NavigationWrapper = styled.section`
```

Comment:
it is a little bit confusing that you have NavigationWrapper here and another component with the same name in the same directory

Reply:
very true, will change that